pandas/tests/reshape/merge/test_merge_antijoin.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+import pandas as pd
+from pandas import (
+    DataFrame,
+    MultiIndex,
+)
+import pandas._testing as tm
+from pandas.core.reshape.merge import merge
+
+
+class TestMergeAntiJoin:
```

Comment:
Nit: Could you drop this test class? Ideally the test suite should just be functional tests